### ISSUE TITLE
Establish working python environment

### ISSUE THREAD

**Issue by e-lo** [BayAreaMetro/tm2py#2]
As a tm2py user and developer, I need a documented "working python environment" in order to have the same python environment for EMME, network_wrangler, Lasso, and tm2py and not run into issues. 

### Outcome

- [x] `requirements.txt` which passes tests
- [x] `environment.yml` which passes tests
- [x] `readme.md` section which describes any installation issues

### Relevant background

- [Wrangler reqs](https://github.com/wsp-sag/network_wrangler)
- [Lasso reqs](https://github.com/wsp-sag/Lasso)
- EMME reqs [ cc: @inrokevin to add in comments ]
- [TM3 Requirements Doc't ](https://docs.google.com/document/d/1Zx3aKg4olK_w9DqpjaRUJcfb_R78tozGyTtR-XDy9OY/edit?usp=sharing) (a restricted doc't)

### Tests
Run:  `modeler_support` to make sure EMME works with environment

cc: @inrokevin in case I need support in running tets

**Comment by e-lo** [comment]
From @inrokevin:
> Below is a complete list of dependencies for the current Emme 4.5.0 version (distributed and compatible with Python 3.7). Note that other versions may also be compatible but are not tested. 

```
appdirs==1.4.4
apsw==3.27.2.post1
attrs==19.1.0
autobahn==0.8.14
Automat==0.7.0
backcall==0.1.0
bleach==3.1.0
Cheetah3==3.2.1
cheroot==8.2.1
Click==7.0
click-plugins==1.0.4
cligj==0.5.0
codegen==1.0
colorama==0.4.1
colour==0.1.5
constantly==15.1.0
cycler==0.10.0
decorator==4.4.0
defusedxml==0.6.0
distlib==0.3.1
entrypoints==0.3
filelock==3.0.12
Fiona==1.8.5
GDAL==2.3.3
hyperlink==18.0.0
idna==2.8
importlib-metadata==3.7.0
incremental==17.5.0
ipykernel==4.10.0
ipython==7.9.0
ipython-genutils==0.2.0
ipywidgets==7.4.2
jaraco.functools==2.0
jedi==0.13.3
Jinja2==2.10
jsonschema==3.0.1
jupyter==1.0.0
jupyter-client==5.3.4
jupyter-console==6.0.0
jupyter-core==4.6.1
kiwisolver==1.1.0
lxml==4.4.1
MarkupSafe==1.1.1
matplotlib==2.2.4
mistune==0.8.4
mock==3.0.5
more-itertools==7.2.0
munch==2.3.2
nbconvert==5.4.1
nbformat==4.4.0
notebook==6.0.2
numexpr==2.7.1
numpy==1.16.2
OpenMatrix==0.3.4.1
orderedset==2.0.1
ortools==7.3.7083
pandas==0.24.2
pandocfilters==1.4.2
parso==0.4.0
pickleshare==0.7.5
ply==3.11
pml==1.20
prometheus-client==0.6.0
prompt-toolkit==2.0.10
protobuf==3.11.2
PuLP==1.6.9
Pygments==2.3.1
PyHamcrest==1.9.0
PyOpenGL==3.1.5
pyparsing==2.4.0
pyproj==1.9.6
pyrsistent==0.15.2
pyshp==2.1.0
python-dateutil==2.8.0
pytz==2018.9
pywin32==224
pywinpty==0.5.5
pyzmq==18.0.1
qtconsole==4.4.4
Send2Trash==1.5.0
Shapely==1.6.4.post1
six==1.12.0
spark-parser==1.8.9
tables==3.5.1
terminado==0.8.1
testpath==0.4.2
tornado==5.1.1
traitlets==4.3.2
Twisted==18.9.0
typing-extensions==3.7.4.3
uncompyle6==3.7.4
virtualenv==20.4.2
wcwidth==0.1.7
web.py===0.40-16-g4504b38
webencodings==0.5.1
widgetsnbextension==3.4.2
xdis==5.0.6
zipp==3.4.0
zope.interface==4.6.0
```

**Comment by e-lo** [comment]
Added to `requirements.txt` in https://github.com/BayAreaMetro/tm2py/commit/f80e1482021e1dbb0c7fe8d10040a6b3493fcf2d

Wondering if @inrokevin can test with EMME (my access to a server w/ a license  is enqueue) ?

**Comment by e-lo** [comment]
`environment.yml` added in #12

**Comment by e-lo** [comment]
Closed via #16
